scripts/dev.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
+set -eux
 
-pip install -r requirements.txt
-pip install -r dev_requirements.txt
-pip install -e .
+pip install flit
+flit install --symlink
```

Comment:
Why need to symlink? The assumption is that you change something and then you have another project that uses the local implementation? So the symlink will automatically allow the other project to use the latest changes?

Reply:
>  So the symlink will automatically allow the other project to use the latest changes

Yes. This is similar to running `pip install -e`. But by right dev should run this inside a virtualenv.

We need this so that we can run the tests since it needs to import `pyfreedb.*`